doc/tutorial/faq.txt

```diff
@@ -0,0 +1,68 @@
+.. _faq:
+
+===========================
+Frequently Asked Questions
```

Comment:
"Frequently Asked Questions" does not appear, and "How to update..." appears as the title.